src/utils/parseResources.js

```diff
@@ -4,15 +4,28 @@ import logger from './logger';
 import isArrayOfStrings from './isArrayOfStrings';
 import flattenArray from './flattenArray';
 
+const getGlobPaths = (path) => {
+  const files = glob.sync(path);
+
+  if (files.length === 0) {
+    throw new Error(`
+      Couldn't find any files with the glob ${path}.
+      Did you forget to resolve this to an absolute path with path.resolve?
+    `);
+  }
+
+  return files;
+};
+
 export default locations => {
```

Comment:
Hi @Tomburgs how do you feel about this refactoring for clarity?

This is called with:

```js
  const resourcesLocations = parseResources(resourcesFromConfig);
```

That's OK. It's either an array or list of patterns


and then that var becomes `locations` currently

and this function `getGlobPaths` takes singular `path`

Maybe we could change the function to take `pattern`

```js
const getGlobPaths = (pattern) => {
```

this file changes to take `patterns` for the argument
```js
export default patterns => {
```

And change this file and the calling reference to it to 
`expandResourceGlobs`

https://github.com/shakacode/sass-resources-loader/blob/ea22038a92/src/loader.js#L34

```js
  const resourcesLocations = expandResourceGlobs(resourcesFromConfig);
```